src/lib/CONFIG/config.cpp

```diff
@@ -1023,6 +1023,8 @@ RxConfig::SetDefaults(bool commit)
     SetPwmChannel(2, 0, 2, false, 0, false); // ch2 is throttle, failsafe it to 988
 #endif
 
+    m_config.teamraceChannel = AUX7; // CH11
```

Comment:
This forces a hard coded AUX??? 😱

Reply:
Oh maybe this is just the default starting value?

Reply:
If it is just the default... maybe add a comment saying as much?

Reply:
It is hard to see because the diff doesn't show enough context, but the entire function is setting defaults-- it is `RxConfig::SetDefaults()`.